### DISCUSSION TITLE
Did Anyone know how to build a docker that compatible with RAPIDS?

### DISCUSSION THREAD

**Discussion by NicoNicoNico123** [pycaret/pycaret#3767] [Q&A]
I struggle to how to install them in 1 container as it's always cause error during build docker, it would be great if someone share the dockerfile

**Comment by jmakov** [comment]
Tried a couple of Weeks ago now that they switched to non root users. Had to dig deep into their repos to see how they actually are doing it. They're offering what they call dev containers for cases where you want to install additional packages. Had a working solution (for a single user), but ditched it for a simple conda env.
